Dockerfile

```
@@ -10,6 +10,8 @@ COPY . .
 RUN yarn install --non-interactive --frozen-lockfile
 
 RUN node ./scripts/getGitData /usr/app/.git-data.json
```

```suggestion
RUN node ./scripts/getGitData /usr/app/.git-data.json && rm -r .git
```
to avoid another docker layer